src/main/kotlin/org/jitsi/jibri/selenium/status_checks/MediaReceivedStatusCheck.kt

```diff
@@ -52,15 +54,25 @@ class MediaReceivedStatusCheck(
     }
 
     companion object {
+        private val noMediaTimeout: Duration by config {
```

Comment:
Same comments as above (no old config, default in `reference.conf` instead of here, make the path `jibri.call-status-checks.no-media-timeout`)